core/commands/snapshot_command.rb

```diff
@@ -277,9 +286,10 @@ def revert_snapshot(node_name, snapshot_name)
       run_reliable_command("virsh snapshot-revert --domain #{@nodes_directory_name}_#{node_name} --snapshotname #{full_snapshot_name}")
       pwd = Dir.pwd
       Dir.chdir @path_to_nodes
-      run_reliable_command("vagrant ssh #{node_name} -c '/usr/bin/sudo service ntpd stop'")
+      ntp_service = ntp_service_name(box_by_node_name(node_name))
+      run_reliable_command("vagrant ssh #{node_name} -c '/usr/bin/sudo /bin/systemctl stop #{ntp_service}.service'")
       run_reliable_command("vagrant ssh #{node_name} -c '/usr/bin/sudo ntpdate 0.europe.pool.ntp.org'")
```

Comment:
The NtpDate is deprecated. http://support.ntp.org/bin/view/Dev/DeprecatingNtpdate